test/unit/TestIBMDriver.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2015 Spotify AB. All rights reserved.
+#
+# The contents of this file are licensed under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with the
+# License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+import unittest
+
+from napalm.ibm import IBMDriver
+from base import TestNetworkDriver
+
+
+class TestIOSXRDriver(unittest.TestCase, TestNetworkDriver):
```

Comment:
@thomas-mangin you are right. copy and past will kill me someday ;) Thanks for letting me know :)